rca/account_management/views.py

```diff
@@ -119,6 +125,37 @@ def form_valid(self, form):
             student_page.save_revision()
             student_page.unpublish()
 
+            # Create a specific group for this student so they have edit access to their page
+            # and their image collection
+            specific_student_group, created = Group.objects.get_or_create(
+                name=f"Student: {form.cleaned_data['first_name']} {form.cleaned_data['last_name']}"
+            )
+
+            # Create new add GroupPagePermission
+            GroupPagePermission.objects.get_or_create(
+                group=specific_student_group, page=student_page, permission_type="edit"
+            )
+
+            # Create new GroupCollectionPermission for Profile Images collection
+            GroupCollectionPermission.objects.get_or_create(
+                group=specific_student_group,
+                collection=Collection.objects.get(
+                    name=form.cleaned_data["student_user_image_collection"]
+                ),
+                permission=Permission.objects.get(codename="add_image"),
+            )
+            GroupCollectionPermission.objects.get_or_create(
+                group=specific_student_group,
+                collection=Collection.objects.get(
+                    name=form.cleaned_data["student_user_image_collection"]
+                ),
+                permission=Permission.objects.get(codename="choose_image"),
+            )
```

Comment:
remove `get` here - it's masking potential bugs - this journey would always be creating new groups